evap/evaluation/management/commands/send_reminders.py

```diff
@@ -29,7 +29,7 @@ def handle(self, *args, **options):
 
         for recipient in recipients:
             due_courses = dict()
-            for course in Course.objects.filter(participants=recipient, state='in_evaluation').exclude(voters=recipient):
+            for course in Course.objects.filter(participants=recipient, state='in_evaluation', vote_end_date__in=check_dates).exclude(voters=recipient):
```

Comment:
I guess students simply got reminded to vote on all their unvoted courses, as long as one of them was ending in REMIND_X_DAYS_AHEAD_OF_END_DATE.  Do we want to keep that behavior?